timestamp,category,user_id,function,line_number,flag,message,exception
2024-06-08 12:08:17,ERROR,asif24,create_or_pass_folder,197,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6cf04cb-001e-0054-086e-b99f8c000000
Time:2024-06-08T06:38:15.8565706Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6cf04cb-001e-0054-086e-b99f8c000000
Time:2024-06-08T06:38:15.8565706Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 12:08:17,ERROR,asif24,create_or_pass_folder,206,0,"Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6cf04cb-001e-0054-086e-b99f8c000000
Time:2024-06-08T06:38:15.8565706Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6cf04cb-001e-0054-086e-b99f8c000000
Time:2024-06-08T06:38:15.8565706Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6cf04cb-001e-0054-086e-b99f8c000000
Time:2024-06-08T06:38:15.8565706Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6cf04cb-001e-0054-086e-b99f8c000000
Time:2024-06-08T06:38:15.8565706Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6cf04cb-001e-0054-086e-b99f8c000000
Time:2024-06-08T06:38:15.8565706Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:f6cf04cb-001e-0054-086e-b99f8c000000
Time:2024-06-08T06:38:15.8565706Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 12:08:17,INFO,asif24,home,1380,1,User asif24 logged in successfully,
2024-06-08 12:08:17,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:08:17,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:08:17,ERROR,asif24,create_or_pass_folder,197,0,Function create_or_pass_folder error,"Function create_or_pass_folder error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:2b1a839e-c01e-000d-106e-b99a0a000000
Time:2024-06-08T06:38:16.6153014Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:2b1a839e-c01e-000d-106e-b99a0a000000
Time:2024-06-08T06:38:16.6153014Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 12:08:17,ERROR,asif24,create_or_pass_folder,206,0,"Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:2b1a839e-c01e-000d-106e-b99a0a000000
Time:2024-06-08T06:38:16.6153014Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:2b1a839e-c01e-000d-106e-b99a0a000000
Time:2024-06-08T06:38:16.6153014Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> ","Function create_or_pass_folder Error creating folder'asif24': Value for one of the query parameters specified in the request URI is invalid.
RequestId:2b1a839e-c01e-000d-106e-b99a0a000000
Time:2024-06-08T06:38:16.6153014Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:2b1a839e-c01e-000d-106e-b99a0a000000
Time:2024-06-08T06:38:16.6153014Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error> 
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 190, in create_or_pass_folder
    container_client.get_blob_client(container_name, user_folder).upload_blob("""")
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\core\tracing\decorator.py"", line 78, in wrapper_use_tracer
    return func(*args, **kwargs)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_blob_client.py"", line 765, in upload_blob
    return upload_block_blob(**options)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_upload_helpers.py"", line 195, in upload_block_blob
    process_storage_error(error)
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\venv\lib\site-packages\azure\storage\blob\_shared\response_handlers.py"", line 184, in process_storage_error
    exec(""raise error from None"")   # pylint: disable=exec-used # nosec
  File ""<string>"", line 1, in <module>
azure.core.exceptions.HttpResponseError: Value for one of the query parameters specified in the request URI is invalid.
RequestId:2b1a839e-c01e-000d-106e-b99a0a000000
Time:2024-06-08T06:38:16.6153014Z
ErrorCode:InvalidQueryParameterValue
queryparametername:snapshot
queryparametervalue:asif24
Content: <?xml version=""1.0"" encoding=""utf-8""?>
<Error><Code>InvalidQueryParameterValue</Code><Message>Value for one of the query parameters specified in the request URI is invalid.
RequestId:2b1a839e-c01e-000d-106e-b99a0a000000
Time:2024-06-08T06:38:16.6153014Z</Message><QueryParameterName>snapshot</QueryParameterName><QueryParameterValue>asif24</QueryParameterValue></Error>"
2024-06-08 12:08:17,INFO,asif24,home,1380,1,User asif24 logged in successfully,
2024-06-08 12:08:18,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:08:20,INFO,asif24,get_data_source,1984,1,table_update route successfully send data,
2024-06-08 12:08:41,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:08:42,INFO,asif24,get_data_source,1984,1,table_update route successfully send data,
2024-06-08 12:08:54,INFO,asif24,upload_to_blob,255,1,Function upload_to_blob successfully created blob_client.url,
2024-06-08 12:08:54,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:08:54,INFO,asif24,popup_form,1504,1,Data Uploaded Successfully,
2024-06-08 12:08:54,INFO,asif24,get_data_source,1984,1,table_update route successfully send data,
2024-06-08 12:09:04,INFO,asif24,Cogni_button,1687,1,CogniLink load button pressed,
2024-06-08 12:09:12,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:09:26,INFO,asif24,update_bar_chart_from_blob,317,1,Function update_bar_chart_from_blob successfully Return blob_list,
2024-06-08 12:09:29,INFO,asif24,update_when_file_delete,579,1,Function update_when_file_delete Data Loaded Successfully,
2024-06-08 12:09:29,INFO,asif24,Cogni_button,1699,1,Load Data Function executed in 24.779727458953857 seconds,
2024-06-08 12:09:29,INFO,asif24,get_data_source,1984,1,table_update route successfully send data,
2024-06-08 12:10:51,INFO,asif24,perform_lda___Q_A,1006,1,QnA topics generated.,
2024-06-08 12:10:51,INFO,asif24,handle_ask_question,1870,1,Answer Generated in 7.8671464920043945 seconds,
2024-06-08 12:11:04,ERROR,asif24,handle_clear_chat,1912,0,clear_chat for ask_question route error,"clear_chat for ask_question route error
Traceback (most recent call last):
  File ""C:\Users\Asif\Desktop\New_version_CogniLink\app.py"", line 1893, in handle_clear_chat
    embeddings_list = result['documents']
KeyError: 'documents'"
